live-examples/css-examples/basic-user-interface/outline-offset.html

```diff
@@ -0,0 +1,37 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="outline-offset">
+
+    <div class="example-choice">
+        <pre><code id="example_one" class="language-css">outline-offset: 4px;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_two" class="language-css">outline-offset: 0.5rem;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_three" class="language-css">outline-offset: 1em;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_four" class="language-css">outline-offset: 1ch;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
```

Comment:
I'm wondering, instead of having four examples that simply uses different units, perhaps you can have the first two, and then a third that is based on the example here? https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset#example

You probably only need to use the top two lines, i.e. 

```
outline: 1px dashed red;
outline-offset: 10px;
```